syncapi/streams/stream_pdu.go

```diff
@@ -346,6 +348,46 @@ func (p *PDUStreamProvider) addRoomDeltaToResponse(
 	// Now that we've filtered the timeline, work out which state events are still
 	// left. Anything that appears in the filtered timeline will be removed from the
 	// "state" section and kept in "timeline".
+
+	for i, ev := range events {
```

Comment:
This feels like a horrible hack atm, but this way we can at least invite other users to the room.
Changing powerlevels currently results in a weird behavior (alice made bob a moderator):

> alice changed the power level of alice from Default to Admin, bob from Default to Moderator, BzmJ3YUq9n1c5A8ZHTNSKmQNGNL8WT8q1sowxNd9XpE from Admin to Default.

Also changing the powerlevels afterwards is not possible anymore.